src/persist/src/indexed/encoding.rs

```diff
@@ -65,8 +65,7 @@ pub enum BatchColumnarFormat {
 impl BatchColumnarFormat {
     /// Returns a default value for [`BatchColumnarFormat`].
     pub const fn default() -> Self {
-        // IMPORTANT: Default to only writing Row data and not our newer structured format.
-        BatchColumnarFormat::Row
+        BatchColumnarFormat::Both(2)
```

Comment:
@bkirwi / @ParkMyCar

Reply:
Personally I would opt to keep this at `BatchColumnarFormat::Row`, but Ben is driving the rollout so will defer to him!

Reply:
Yeah, this isn't fully rolled out yet - let's hold off on this.